app/src/main/kotlin/maru/app/MaruAppCli.kt

```diff
@@ -36,29 +53,57 @@ class MaruAppCli : Callable<Int> {
     arity = "1..*",
     required = true,
   )
-  private val configFiles: List<File>? = null
+  lateinit var configFiles: List<File>
 
-  @CommandLine.Option(
-    names = ["--maru-genesis-file"],
-    paramLabel = "BEACON_GENESIS.json",
-    description = ["Beacon chain genesis file"],
-    required = true,
+  @CommandLine.ArgGroup(multiplicity = "1", exclusive = true, validate = true)
+  lateinit var genesisOptions: GenesisOptions
+
+  class GenesisOptions(
+    @CommandLine.Option(
+      names = ["--maru-genesis-file"],
```

Comment:
I think this can be just `--genesis-file` no need for the maru prefix

Reply:
I'd not recommend this for now, as this would break backward-compatibilty for our external partners, unless we really want to do it and have a plan to communicate clearly to current external partners

Reply:
Fair enough, better to leave as is then

Reply:
I think we can take take the same strategy as with linea.l1EthRpc [here](https://github.com/Consensys/maru/pull/451#discussion_r2466067069)

Meaning that we'd accept both `maru-genesis-file` and `genesis-file` and maybe even print a warning that the former is deprecated